app.js

```diff
@@ -47,35 +47,35 @@ var requirementObj = {
 };
 
 var reqStatus = {
-  "Requested":	1,
-  "Evaluated":	7,
-  "Accepted":	5,
-  "Rejected":	6,
-  "Planned":	2,
-  "In Progress":	3,
-  "Developed":	4,
-  "Obsolete":	8,
-  "Tested":	9,
-  "Completed":	10,
-}
+  "Requested": 1,
+  "Evaluated": 7,
+  "Accepted": 5,
+  "Rejected": 6,
+  "Planned": 2,
+  "In Progress": 3,
+  "Developed": 4,
+  "Obsolete": 8,
+  "Tested": 9,
+  "Completed": 10,
+};
 
 //column ranges for different sheets, currently only requirements.
 var columnRanges = {
   requirements: "A3:K",
   customFieldRanges: {
-  requirements: ["N", "AQ"],
-}
+    requirements: ["N", "AQ"],
+  }
 };
 
 //array to hold custom field names
 var customFieldNames = [];
 
 function cleanObject(Obj) {
```

Comment:
This is a common theme throughout but a lot of these functions need a quick comment or two. I think they're clearly written but a quick sentence explaining what it does would add up to quite a bit of time saved reading through this application!